contracts/Bounty.sol

```diff
@@ -26,4 +26,12 @@ contract Bounty {
 	function kill(address payout) {
 		selfdestruct(payout);
 	}
-}
\ No newline at end of file
+
+	function setPayee(address newPayee) {
+		payee = newPayee;
+	}
+
+	function setGithubDetails(GithubDetails newGithubDetails) {
```

Comment:
Why would we need to change the githubdetails after creation?

Reply:
Eeeerm, I think I put that in while refactoring to try and fix a different error, but you're right, doesn't look like we need it.